codegen/test_data/endpoint_tests/bar_bar_method_argwithheaders_test.gogen

```diff
@@ -0,0 +1,24 @@
+// Code generated by zanzibar
```

Comment:
Seems the code generation has issues. These are empty files.

Reply:
these are files without tests added, we should 1: skip if no tests fixtures 2: enforce every endpoint should have test fixtures, I vote for 2 :)